src/main/config.js

```diff
@@ -92,6 +93,7 @@ function init(argv) {
     homeDir = electronApp.getPath('home');
     userDataDir = electronApp.getPath('userData');
     desktopDir = electronApp.getPath('desktop');
+    ubuntuDesktopDir = path.join(homeDir, '.local/share/applications');
```

Comment:
```suggestion
    ubuntuDesktopDir = path.join(homeDir, '.local', 'share', 'applications');
```
Though, it's not a big problem on ubuntu, it's better to avoid writing slash so that it reduces the chance to cause path problem.